package.json

```diff
@@ -66,5 +66,8 @@
   },
   "ember": {
     "edition": "octane"
+  },
+  "dependencies": {
+    "glob": "^7.1.6"
```

Comment:
Where is this coming from? 🤔

Reply:
\"lint:!(fix)\" part will be resolved by glob only

Reply:
[Glob-package](https://www.npmjs.com/package/glob)